src/expanded-date-picker.ts

```diff
@@ -105,15 +114,28 @@ export class ExpandedDatePicker extends LitElement {
     });
     this.dispatchEvent(event);
     this.closeModal();
+
+    const yearQuery =
+      this.maxSelectedDate && this.minSelectedDate
+        ? `year:[${this.minSelectedDate} TO ${this.maxSelectedDate}]`
+        : undefined;
+
+    this.analyticsHandler?.sendEvent({
+      category: analyticsCategories.default,
+      action: analyticsActions.histogramChangedFromModal,
+      label: yearQuery,
```

Comment:
QUESTION: Often, we put the page's URL in the label. Would it be more useful to put the URL or the collection identifier here?

Reply:
Yeah, I agree that's more helpful. Both events now use the URL (which will include any collection identifier anyway).